api/user.json

```diff
@@ -154,32 +148,30 @@
     "climateActionIncentiveIsRural": null
   },
   "refund": {
-    "Total Payable": {
+    "totalPayable": {
       "name": "Total Payable",
       "line": 435,
-      "amount": 0.00
+      "amount": 1593.00
     },
-    "Total Deducted": {
+    "totalDeducted": {
       "name": "Total Deducted",
       "line": 437,
-      "amount": 0.00
+      "amount": 2312.88
     },
-    "Total Credits": {
+    "totalCredits": {
       "name": "Total Credits",
-      "line": 482,
       "amount": 2500.00
     },
-    "Total Refund": {
+    "totalRefund": {
       "name": "Total Refund",
       "line": 484,
-      "amount": 3200.00
+      "amount": 719.88
     }
   },
   "benefits": {
-    "Canada child benefit": 6639.00,
-    "Climate action incentive": 0.00,
-    "GST/HSTC": 0.00,
-    "Trillium": 300.00,
-    "totalBenefits": 6939.00
+    "Climate action incentive": 154.00,
+    "GST/HSTC": 440.00,
+    "Ontario trillium benefits": 307.92,
```

Comment:
I think the correct words are "Ontario Trillium Benefit"

https://www.ontario.ca/page/ontario-trillium-benefit